packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -636,10 +638,7 @@ abstract class BaseMethod implements PythonBase {
     }
 
     if (decorators.length > 0) {
-      // "# type: ignore[misc]" needed because mypy does not know how to check decorated declarations
```

Comment:
Is this removed because of the PR that introduces pyright?

Reply:
It appears mypy is now able to check most of these.